cmd/root.go

```diff
@@ -46,6 +46,7 @@ func ServerFlags(flags *pflag.FlagSet) {
 	flags.DurationVar(&rules.Period, "rules-period", 5*time.Minute, "Period to run the rules")
 	flags.BoolVar(&disablePostgrest, "disable-postgrest", false, "Disable postgrest")
 	flags.StringVar(&mail.FromAddress, "email-from-address", "no-reply@flanksource.com", "Email address of the sender")
+	flags.StringVar(&db.PostgresDBAnonRole, "postgrest-anon-role", "postgres_anon", "Postgres anonymous role")
```

Comment:
Just for backward compatibility, I think the default value should be `postgrest_anon` since I assume most people have that user setup already

Reply:
Yeah, else we have to make changes in duty and the helm charts